src/runtime/propensity.js

```diff
@@ -28,17 +28,18 @@ export class Propensity {
   /**
    * @param {!Window} win
    * @param {!../model/page-config.PageConfig} pageConfig
-   * @param {!../api/client-event-manager-api.ClientEventManagerApi} eventManager
+   * @param {!./deps.DepsDef} deps
    */
```

Comment:
Propensity class must remain independent of rest of SwG boot strapping. Hence the API call to enableLoggingSwgEvents() is necessary to get the configuration.

Reply:
Are you saying that I can't pass in a deps?  I don't understand the second sentence: if it can't have deps then how can enableLoggingSwgEvents() get the configuration?  Or is something else supposed to call it when the configuration changes?